src/Select.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { theme } from 'styled-system'
+
+import { Box, Flex, Icon, Label } from '../src'
+
+const parseChevronIcon = isExpanded => {
+  const ChevronIcon = isExpanded
+    ? Icon.extend`
+        transform: rotate(180deg);
```

Comment:
We should use the `chevronUp` and `chevronDown` icons here instead of rotating – the icons might be optically aligned, rather than mathematically centered

Reply:
Sounds good. I used transform rotation because I was observing different between `chevronUp` and `chevronDown`. I wanted to avoid the graphic difference for the icons.

Updated to use `chevronUp` for the `isExpanded` style. Will check with the designer to assess the icon difference to see if there's a real concern.